.github/renovate.json5

```
@@ -8,7 +8,32 @@
   ignorePaths: [
     "**/tests/**",
   ],
+  regexManagers: [
+    {
+      fileMatch: [
+        '^Cargo.toml$',
+      ],
+      matchStrings: [
+        'rust-version.*?(?<currentValue>\\d+\\.\\d+(\\.\\d+)?)',
```

```suggestion
        'rust-version\\s*=\\s*"(?<currentValue>\\d+\\.\\d+(\\.\\d+)?)"',
```

I would probably go with this slightly more specific regex, but I think it should work either way :)

I'm using something roughly similar in https://github.com/Turbo87/renovate-config/blob/04f6fb54cca78973deacec582fb36b4bf5f2da9c/rust/updateToolchain.json#L8